### ISSUE TITLE
Ping

### ISSUE THREAD

**Issue by Leo-MapCreator** [haxball/haxball-issues#281]
Why the ping is not affect by high values? My ping graph looked like a 1500*cosine curve and my ping was 94, wtf? Create a secondary number that measures ping consistency and make it possible for others user to see this number too, this way we can prove that we did in fact lag and that we are just not making excuses for missing the ball

**Comment by Leo-MapCreator** [comment]
I was thinking about this when the room i was lagged, you can just do standard deviation with the last 5 values or even last 10

**Comment by Leo-MapCreator** [comment]
Or maybe the amplitude of the last 5

**Comment by cr0ndor** [comment]
I agree. Should be reverted to flash standard. As it stands, it's just useful for determining if someone is faking another player.
